src/main/java/org/akaza/openclinica/service/ImportValidationServiceImpl.java

```
@@ -0,0 +1,395 @@
+                return true;
+            default:
+                itemCountInForm.setInsertedUpdatedSkippedItemCountInForm(itemCountInForm.getInsertedUpdatedSkippedItemCountInForm() + 1);
+                throw new OpenClinicaSystemException(FAILED, ErrorConstants.ERR_ITEM_TYPE_NOT_SUPPORTED);
+        }
+    }
+
+    private Boolean validateCheckBoxOrMultiSelect(ResponseSet responseSet, String value) {
+        String[] values = value.split(",");
+
+        for (String v : values) {
+            if (!responseSet.getOptionsValues().contains(v)) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    private Boolean validateRadioOrSingleSelect(ResponseSet responseSet, String value) {
+        if(responseSet.getOptionsText().equals("_") && responseSet.getOptionsValues().equals("_"))
+            return true;
+        if (!responseSet.getOptionsValues().contains(value)) {
+            return false;
+        }
+        return true;
+    }
+
+    private Boolean validateItemDataType(Item item, String value, ItemCountInForm itemCountInForm) {
+        ItemDataType itemDataType = item.getItemDataType();
+        switch (itemDataType.getCode()) {
+            case "BL":
+                return validateForBoolean(value);
+            case "ST":
+                return true;
+            case "INT":
+                return validateForInteger(value);
+            case "REAL":
+                return validateForReal(value);
+            case "DATE":
+                return validateForDate(value);
+            default:
+                itemCountInForm.setInsertedUpdatedSkippedItemCountInForm(itemCountInForm.getInsertedUpdatedSkippedItemCountInForm() + 1);
+                throw new OpenClinicaSystemException(FAILED, ErrorConstants.ERR_ITEM_TYPE_NOT_SUPPORTED);
+        }
+    }
+
+    private Boolean validateForBoolean(String value) {
+        if (!value.equals("true") && !value.equals("false")) {
+            return false;
```

Instead of returning false throw the errror